MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2246,6 +2240,18 @@ public Person findBestInRole(PersonnelRole role, String skill) {
         return findBestInRole(role, skill, null);
     }
 
+    public @Nullable Person findBestAtSkill(String skill) {
```

Comment:
Just a heads up, we're moving towards having everything JavaDoc'd, obviously not necessary while this is a WIP but something you'll likely need to address before it gets merged. We're aware the majority of our stuff isn't currently JavaDoc'd, so that's something we're trying to address. So if you create a new Class, Method, or edit an existing Class or Method please add JavaDoc's*

*This is going to be added to the automated tests at some point